### PR TITLE
user model done, except the app object(I don't know the layout), mong…

### PR DESCRIPTION
**Pull request by milos381**
…oose model, REST CRUD, grabbing the data from DB, only thing left is graphQL integration

### CODE REVIEW

**Review by shalarewicz — CHANGES_REQUESTED** [review]
Issue #20 should be resolved and merged into this branch prior to approval. 

The PR title and description should be updated to provide the issues this PR resolves, a concise summary of the changes as well as evidence of the changes working as expected.

File: .eslintrc.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
         "airbnb", 
         "airbnb/hooks", 
         "airbnb-typescript", 
-        "prettier"
+        "prettier",
+        "react-app"
```
Comment (shalarewicz) [inline .eslintrc.json:13]: Heads up `"airbnb"` has support for react so `"react-app"` is going to overwrite a lot of those.

**Review by shalarewicz — APPROVED** [review]
Only comment is on "react-app"  in the eslint config.

File: .eslintrc.json (hunk 1 above)
Comment (feiw101) [reply .eslintrc.json:13]: Just removed.

**Review by feiw101 — APPROVED** [review]

**Pull request merged**